dspace-api/src/main/java/org/dspace/app/launcher/Readme.md

```diff
@@ -0,0 +1,57 @@
+# Adding new scripts to the dspace CLI 
+
+Scripts can be added to the script launcher using 2 mechanisms
+
+1. Defining a script in [dspace]/config/launcher.xml
+2. Creating a spring bean of type _org.dspace.app.launcher.CommandType_
+
+The advantage of the second mechanism is that a dependency can provide both the script and add it to the launcher without needing to modify the configuration of the receiving party.
+
```

Comment:
This is a very helpful documentation page.  I would also recommend adding this content to a page on the DSpace wiki.  Perhaps on a new page under https://wiki.duraspace.org/display/DSDOC6x/DSpace+Development